source/appModules/outlook.py

```diff
@@ -133,7 +133,15 @@ def _get_outlookVersion(self):
 		return outlookVersion
 
 	def isBadUIAWindow(self,hwnd):
-		if winUser.getClassName(hwnd) in ("WeekViewWnd","DayViewWnd"):
+		windowClass=winUser.getClassName(hwnd)
+		# #2816: Outlook versions before 2016 auto complete does not fire enough UIA events, IAccessible is better.
+		if windowClass=="NetUIHWND":

```

Comment:
There are a few usages of this "NetUIHWnd" can you put them in a named constant, perhaps `outlookWindowClass`

Reply:
I don't believe this would improve things. A window class string is as gooder constant from my point of view.  Something like OutlookWindowClass does not addiquitly describe the string as this NetUIHWND class is used in more than just Outlook. It seems to be some kind of generic window class for Ribbons. But to be clear, not the only one though, there is also NetUIToolWindow. Personally I think the strings are good enough. However, I guess we could convert them in to constants I.e. `NetUIHWND="NetUIHWND"` but I really don't see the point.

Reply:
Ok. My hope was that we could describe the string better. As someone unfamiliar with the value, it doesn't tell you any of the things you just mentioned. Having a constant with comment you just wrote would be helpful in my opinion. Ideally a name that describes it well, it seems like that is difficult.